cibyl/plugins/openstack/deployment.py

```diff
@@ -219,3 +234,5 @@ def merge(self, other):
             self.add_node(node)
         for service in other.services.values():
             self.add_service(service)
+        if not self.stages.value and other.stages.value:
```

Comment:
Would it not be easier to just add the two lists up?

Reply:
Honestly, I'm not sure what would be the best behaviour here. Just adding the two lists would probably end up with duplicates. Furthermore, the stages should probably have their order preserved to provide a more accurate picture and assuming that I was not able to come up with any good way of mixing the two lists.